news/templates/news/news_base_detail.html

```diff
@@ -19,9 +20,19 @@
 {% block body %}
     <div class="news-obj {% block extra_classes %}{% endblock extra_classes %}">
         <div id="sticky" class="ui text container basic segment">
-            <img class="news-obj-image {% if news_obj.contain %}contain{% endif %}"
-                 src="{{ news_obj.image.url }}" alt="{{ news_obj.image_description }}"/>
-
+            <div class="ui fluid image">
+                {% if news_obj.private %}
+                    <p class="ui blue make-bg-blue ribbon label primary enlarged">
+                        {% if news_obj.event_type %}
+                            {% translate "internal"|capfirst context "neuter" %}
+                        {% else %}
+                            {% translate "internal"|capfirst context "masculine" %}
+                        {% endif %}
+                    </p>
+                {% endif %}
+                <img class="news-obj-image event {% if news_obj.contain %}contain{% endif %}"
```

Comment:
The event class was added because while working with the internal news labels, I obviously made a change that caused the space between the picture and the admin panel button on the event detail page to disappear. But I have removed it, and instead added a margin-bottom to the news-obj-image class.